examples/demo_ot_detector.ipynb

```diff
@@ -0,0 +1,1403 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We get the bias value for each each of the protected groups - in this case, Male (`0`) and Female (`1`). \n",
+    "\n",
+    "These values range from 0 to 1 and can be interpreted as the difference in percent between the ground truth distribution and the distribution of the protected group: for example, a value of 0.3 would mean a 30% difference."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Measuring bias with respect to `age_cat`\n",
+    "\n",
+    "Now we measure the bias of the same classifier with respect to the age category."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ot_val2 = ot_bias_scan(ground_truth=\"two_year_recid\", classifier=preds, sensitive_attribute='age_cat', data=df, num_iters=1000000)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>age_cat</th>\n",
+       "      <th>ot_val</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>0</td>\n",
+       "      <td>0.000578</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>1</td>\n",
+       "      <td>0.000313</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2</td>\n",
+       "      <td>0.001800</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   age_cat    ot_val\n",
+       "0        0  0.000578\n",
+       "1        1  0.000313\n",
+       "2        2  0.001800"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Total bias = 0.00026458993506953317\n"
+     ]
+    }
+   ],
+   "source": [
+    "bs2 = pd.DataFrame({\"age_cat\": ot_val2.keys(), \"ot_val\": ot_val2.values()})\n",
+    "display(bs2)\n",
+    "\n",
+    "print(\"\\nTotal bias = \", abs(np.diff(list(ot_val2.values()))[0]), sep=\"\")"
```

Comment:
this is just computing the difference between `age_cat` 0 and 1. Is that intentional(/meaningful)?

Reply:
Indeed, this doesn't offer much information in our case, will be removed